lib/zht/pages/ActualPosition.dart

```diff
@@ -48,17 +49,23 @@ class ActualPositionState extends State<ActualPosition>{
       return  Text("LATITUDE: ${_locationData.latitude}, LONGITUDE: ${_locationData.longitude}");
   }
 
-  @override
+ @override
   Widget build(BuildContext context){
-    return new MaterialApp(
-      home: new Scaffold(
-        backgroundColor: Colors.red,
-        body: Center(
-          child: getLatandLng(),
```

Comment:
This page should continue to show the current location AND have a button "make as start"